packages/components/src/card/types.ts

```diff
@@ -71,11 +81,17 @@ type MarginalSubComponentProps = BaseSubComponentProps & {
 	 * @default false
 	 */
 	isBorderless?: boolean;
-	className?: string;
```

Comment:
This is not necessary anymore since the `CardHeader` and `CardFooter` components are using `PolymorphicComponentProps`